### PR TITLE
User_profile custom error handler

### PR DESCRIPTION
**Pull request by rishabh0282**

### CODE REVIEW

File: app/templates/user/user_profile.html [hunk 1]
```diff
@@ -1,3 +1,207 @@
...
+            </div>
+            <div class="col-md-8">
+                <div class="card border-0 shadow rounded-3 mb-3 m-1">
+                    <div class="card-body">
+                        <div class="row">
+                            <div class="col-sm-3">
+                                <h6 class="mb-0">Full Name</h6>
+                            </div>
+                            <div class="col-sm-9 text-secondary">
+                                {{ user.first_name}} {{ user.last_name}}
```
Comment (soumyankar) [inline app/templates/user/user_profile.html:87]: It should be changed to
```
{% if user.user_profile %}
{{ user.user_profile.first_name }}
{% endif %}
```

Wherever there is a call to any of the `user_profile` details, change to `user.user_profile.<detail_name>`

File: app/templates/user/user_settings.html [hunk 2]
```diff
@@ -13,40 +13,43 @@
...
+                src=" https://ui-avatars.com/api/?background=random&length=1&name={{ current_user.username}}" size="48"
+					width="48" height="48" class="rounded-circle me-2">
 				<div class="flex-auto">
 					<h3 class="h3 lh-condensed"> {{ current_user.username }} </h1>
-					<div class="d-flex flex-items-center flex-wrap">
-						<p class="color-text-tertiary mb-0 mr-3"> Your personal account </p>
-					</div>
+						<div class="d-flex flex-items-center flex-wrap">
+							<a class="color-text-tertiary mb-0 mr-3" href="{{ url_for('user_dashboard.user_profile_page', username=current_user.username) }}"> Your personal account </a>
+						</div>
```
Comment (soumyankar) [inline app/templates/user/user_settings.html:27]: I wanted this `a` element to be on the extreme right of this row. Try if you can make that happen, I didnt look too much into it, the code is similar to that of the GitHub profile page

**Review by soumyankar — CHANGES_REQUESTED** [review]

File: app/templates/user/user_settings.html (hunk 2 above)
Comment (rishabh0282) [reply app/templates/user/user_settings.html:27]: Looks better this way

File: app/templates/user/user_profile.html (hunk 1 above)
Comment (rishabh0282) [reply app/templates/user/user_profile.html:87]: I did the change but no luck you do for one I will do the rest

**Pull request merged**